weather@mockturtl/src/3_8/utils.ts

```diff
@@ -75,48 +75,95 @@ export function InjectValues(text: string, weather: WeatherData, config: Config,
+	const tmr = weather.forecasts && weather.forecasts[1] ? weather.forecasts[1] : null;
+	const forecastHours = weather.hourlyForecasts && weather.hourlyForecasts[2] ? weather.hourlyForecasts : null;
+	const tempHourDiff = forecastHours ? ValueChange(TempToUserConfig(forecastHours[0].temp, config, false), TempToUserConfig(forecastHours[2].temp, config, false), false) : "";
+	const conditionTomorrow = tmr ? tmr.condition.main : "";
+	const tempMin = tmr ? TempToUserConfig(weather.forecasts[0].temp_min, config, false).toString() : "";
+	const tempMax = tmr ? TempToUserConfig(weather.forecasts[0].temp_max, config, false).toString() : "";
+	const tempMinTomorrow = tmr ? TempToUserConfig(tmr.temp_min, config, false).toString() : "";
+	const tempMaxTomorrow = tmr ? TempToUserConfig(tmr.temp_max, config, false).toString() : "";
+	const tempsTomorrow = tmr ? TempRangeToUserConfig(tmr.temp_min, tmr.temp_max, config) : "";
+	const tmrMinTempChange = tempMinTomorrow && temp ? SignedNumber(tempMinTomorrow - tempMin) : "";
+	const tmrMaxTempChange = tempMaxTomorrow && temp ? SignedNumber(tempMaxTomorrow - tempMax) : "";
+	const tempsTomorrowWithDifferences = tmr ? tempsTomorrow + " (" + tmrMinTempChange + " / " + tmrMaxTempChange + ")" : "";
+	const sunset = (GetHoursMinutes(weather.sunset, config._show24Hours));
+	const sunrise = (GetHoursMinutes(weather.sunrise, config._show24Hours));
+	const dayLength = ToHoursMinutes(weather.sunset - weather.sunrise);
+	const now = new Date();
+	const sunny = now > weather.sunrise && now < weather.sunset;
+	const daylightRemain = sunny ? ToHoursMinutes(weather.sunset - now) : "";
+	const dayLengthlightRemain = "" + dayLength + (sunny ? " (" + daylightRemain + ")" : "");
+	const valuesPaddingDefaults = [
+	    ['t', temp, 4, true],
+	    ['u', tempUnit],
+	    ['c', condition],
+	    ['c_long', conditionLong],
+	    ['dew_point', dewPoint],
+	    ['humidity', humidity, 3],
+	    ['pressure', pressure, 7],
+	    ['pressure_unit', pressureUnit],
+	    ['extra_value', extraValue],
+	    ['extra_name', extraName],
+	    ['city', city],
+	    ['country', country],
+	    ['search_entry', searchEntry],
+	    ['last_updated', lastUpdatedTime],
+	    ['wind_speed', windSpeed],
+	    ['wind_dir', windDir],
+	    ['wind_arrow', windArrow],
+	    ['wind_deg', windDegree],
+	    ['wind_unit', windUnit],
+	    ['min', tempMin],
+	    ['max', tempMax],
+	    ['tmr_min', tempMinTomorrow],
+	    ['tmr_max', tempMaxTomorrow],
+	    ['tmr_min_diff', tmrMinTempChange],
+	    ['tmr_max_diff', tmrMaxTempChange],
+	    ['tmr_c', conditionTomorrow],
+	    ['tmr_t', tempsTomorrow],
+	    ['tmr_td', tempsTomorrowWithDifferences],
+	    ['sunset', sunset],
+	    ['sunrise', sunrise],
+	    ['day_length', dayLength],
+	    ['day_remain', daylightRemain],
+	    ['day_len_rem', dayLengthlightRemain],
+	    ['t_h_diff', tempHourDiff],
+	    ['br', "\n"]
+	];
+	var regexp, match, pad, padLeft, padChar, newVal, isLiteral, padLiteral, dontPad;
```

Comment:
Use `let`

Reply:
I think that's all up to date, let me know if you spot anything else, cheers :)